src/Authentication/Services/SystemRegisterService.cs

```diff
@@ -44,9 +44,33 @@ public Task<bool> CreateClient(string clientId, CancellationToken cancellationTo
         }
 
         /// <inheritdoc/>
-        public Task<Guid?> CreateRegisteredSystem( RegisteredSystem system, CancellationToken cancellation = default)
+        public Task<Guid?> CreateRegisteredSystem(RegisteredSystem system, CancellationToken cancellation = default)
         {
-            return _systemRegisterRepository.CreateRegisteredSystem(system);
+            int count = system.DefaultRights.Count;
+            string[] defaultRights = new string[count];
+            int i = 0;
+            if (system.DefaultRights != null)
+            {
+                foreach (DefaultRight defaultRight in system.DefaultRights)
+                {
+                    defaultRights[i] = ConvertDefaultRightsToString(defaultRight.Resources);
+                    i++;
+                }
+            }
+
+            return _systemRegisterRepository.CreateRegisteredSystem(system, defaultRights);
+        }
+
+        private static string ConvertDefaultRightsToString(List<AttributePair> pairList)
```

Comment:
we can probably store this as json but we can take this later

Reply:
will do in the next PR next week